lib/capistrano/asg/tasks/rolling.rake

```diff
@@ -139,7 +139,7 @@ namespace :rolling do
       end
     else
       logger.error 'No instances have been launched. Are you using a configuration with rolling deployments?'
-      exit 1
+      raise 'Auto Scaling Group update failed'
```

Comment:
The same as comment above, is there a specific reason to raise instead of exit 1 for this rake task (deploy_test) which does not do any changes to the ASG?